base-server/index.js

```diff
@@ -316,10 +317,12 @@ export class BaseServer {
     this.unbind = []
 
     this.connected = new Map()
+    this.queues = new Map()
     this.nodeIds = new Map()
     this.clientIds = new Map()
     this.userIds = new Map()
     this.types = {}
+    this.queueTypes = []
```

Comment:
Why do we need this array?

Reply:
`queueTypes` holds types of actions to be processed in a queue.  This array is checked in `ClientServer.filter` to see if action should be processed normally or in queue and it is populated if the developer adds true when declaring `type` in a server module.

Reply:
Why not `Map` with `type → queue name`?

Reply:
`Map` would make a lot sense here. But are there any clear advantages over an array?

Reply:
> But are there any clear advantages over an array?

I do not see an advantage of array. Every action has some queue (at least `main`).